tests/test_neon.py

```diff
@@ -9,10 +9,22 @@
 from exo import proc
 from exo.platforms.neon import *
 from exo.stdlib.scheduling import *
+from exo.memory import MemGenError
 
 import numpy as np
 
 
+@pytest.mark.isa("neon")
```

Comment:
This test doesn't actually run anything on NEON, it just needs to get as far as generating C. Therefore it should never be skipped and you can check that `read_neon.c_code_str()` throws the expected `MemGenError`.

Reply:
That makes sense!